src/rgw/rgw_rados.cc

```diff
@@ -9134,7 +9134,7 @@ int RGWRados::check_disk_state(const DoutPrefixProvider *dpp,
     // encode a suggested removal of that key
     list_state.ver.epoch = io_ctx.get_last_version();
     list_state.ver.pool = io_ctx.get_id();
-    cls_rgw_encode_suggestion(CEPH_RGW_REMOVE, list_state, suggested_updates);
+    cls_rgw_encode_suggestion(CEPH_RGW_REMOVE | suggest_flag, list_state, suggested_updates);
```

Comment:
Is `suggest_flag` appropriately named? It seems to be more about whether or not to log the change, perhaps for multisite? If it could be better named, that'd be a nice fix.

Reply:
I suppose it was named "suggest_flag" as it is passed to dir_suggest() and even the flag is called "CEPH_RGW_DIR_SUGGEST_LOG_OP".  Shall I change it to maybe "dir_suggest_log_flag" to be more clear?